stackoverlay.js

```diff
@@ -316,6 +322,12 @@ export class StackOverlay {
         this.setTarget(null);
     }
 
+    touchTriggerPreview() {
+        if (event.type() === Clutter.EventType.TOUCH_BEGIN)
+            return triggerPreview();
+        return Clutter.EVENT_PROPAGATE;
+    }
+
```

Comment:
This can be removed, but the branch protection rules prevent me from doing so

Reply:
hmm, you couldn't delete these lines and then push a new commit?  Unless you were trying to force-push?

Anyways, I see you've created a new PR.

Reply:
I can't even push normally, that's the thing

Reply:
hmm - let me have a look (I might have borked the rules).

Reply:
Yeah, I think I messed up the rules.  You should be good now :+1: